charts/kueue/values.yaml

```diff
@@ -5,6 +5,8 @@ nameOverride: ""
 fullnameOverride: ""
 # Enable each function, like kustomize https://github.com/kubernetes-sigs/kueue/blob/main/config/default/kustomization.yaml
 enablePrometheus: false
+# Enable visibility, set the default to true when the feature is no longer aplha
+enableVisibility: false
```

Comment:
Instead of adding this field, should we extract enabled featuregate from https://github.com/kubernetes-sigs/kueue/blob/d7cb251b16fe4c23cd4f7dc6d38fb68b52663515/charts/kueue/values.yaml#L13-L16?

Reply:
I would like that approach to have a single source of truth, but not sure if this is feasible / how to extract.

Reply:
I guess that we can refer to https://github.com/kubernetes-sigs/kueue/blob/6d57813a52066dab412735deeeb60ebb0cdb8e8e/charts/kueue/templates/_helpers.tpl#L73-L75